iocore/net/QUICNetVConnection.cc

```diff
@@ -463,6 +462,14 @@ QUICNetVConnection::start()
   this->_frame_dispatcher->add_handler(this->_stream_manager);
   this->_frame_dispatcher->add_handler(this->_path_validator);
   this->_frame_dispatcher->add_handler(this->_handshake_handler);
+
+  // regist qlog
+  if (this->_context->config()->qlog_file() != nullptr) {
+    this->_qlog = std::make_unique<QLog::QLogListener>(*this->_context.get(), this->_original_quic_connection_id.hex());
+    this->_qlog->last_trace().set_vantage_point(
+      {"ats", QLog::Trace::VantagePointType::server, QLog::Trace::VantagePointType::server});
+    this->_context->regist_callback(this->_qlog);
```

Comment:
Seems like a nice idea 👍 
I guess we can reuse this mechanism for variety of things (e.g. plugin hooks, metrics, etc.)  in the future.